python_cross_site_scripting/arshad6571_Myform/App.py

```
@@ -0,0 +1,66 @@
+@app.route('/submit', methods=['POST'])
+def submit():
+    email = request.form.get('email')
+    password = request.form.get('password')
+    
+    # Creating a response object
+    response = make_response(f"Received POST request.<br>Email: {email}<br>Password: {password}")
+    response.headers['Custom-Header'] = 'Success'
+    return response
+
+if __name__ == '__main__':
+    app.run(debug=True)
```

Ruff (0.13.3)

Use of `debug=True` in Flask app detected

(S201)